.github/workflows/comptime_ci.yml

```diff
@@ -0,0 +1,51 @@
+name: Time CI
```

Comment:
```suggestion
name: Comptime CI
```
Wouldn't that be a more appropriate name?

Reply:
Indeed, copy-pasta error 😄